src/components/NavigationBar/index.js

```diff
@@ -77,13 +80,13 @@ const UserDetail = styled.div`
     display: None;
   }
 `;
-const CreateDetail = styled.div`
-  color: white;
-  margin-right: 5px;
-  font-size: 1rem;
-  cursor: pointer;
-  bottom: 8px;
-`;
+// const CreateDetail = styled.div`
```

Comment:
Any reasons for commenting? Either remove it or use it

Reply:
I want to remove it but forgot to remove.